aws/cloudformation/moz-single.json

```diff
@@ -245,6 +245,7 @@
             " -e basket_queue_url=",         { "Ref": "FxaBasketAccountChangeQueue" },
             " -e oauth_queue_url=",          { "Ref": "FxaOAuthAccountChangeQueue" },
             " -e profile_queue_url=",        { "Ref": "FxaProfileAccountChangeQueue" },
+            " -e pushbox_queue_url=",        { "Ref": "FxaPushboxAccountChangeQueue" },
```

Comment:
we might be missing this addition in https://github.com/mozilla/fxa-dev/blob/docker/aws/roles/cron_update/tasks/main.yml#L50

Reply:
Yeah, you have to put `pushbox_queue_url={{ pushbox_queue_url }}` in that mess of a line.

Reply:
Aha, thanks, now I know to check ` /var/log/ansible/update.log` for errors too; added